examples/srd_y10_3x2.yaml

```diff
@@ -0,0 +1,260 @@
+generate:
+  augur: # general options
+    # the folowing two options are place-holders for what
+    # will once be part of TJPCov interface
+    fsky : &fsky 0.35 # Y1 - 14,300 sq-deg
+    add_noise: &add_noise False # add a realization of noise?
+    kmax: &kmax 0.3 # kmax in Mpc^-1 for 2pt analyses
+
+  parameters: &parameters
+    Omega_k: 0.0
+    Omega_b: 0.0492
+    Omega_c: 0.26639999999999997  # = 0.3156 - 0.0492
+    w0: -1.0
+    wa: 0.0
+    h: 0.6727
+    sigma8: 0.83
+    n_s: 0.9645
+    transfer_function: 'eisenstein_hu'
+    # IA model
+    eta_ia: 0.0
+    eta_ia_highz: 0.0
+    beta_ia: 1.0
+    A0: 5.0
+
+    mult_bias: 0.0
+
+
+    # Values computed as 0.95/G(z) with fiducial cosmo below:
+
+    {% for i, bias in [(0, 1.08503751), (1, 1.14373505), (2, 1.20458916),
+                       (3, 1.26729764), (4, 1.33159408), (5, 1.39724721),
+                       (6, 1.46405855), (7, 1.53185887), (8, 1.6005048),
+                       (9, 1.66987515)] %}
+    bias_lens{{ i }}: {{ bias }}
+    {% endfor %}
+
+    # photoz shifts for the lensing sources
+    {% for i in range(10) %}
+    src{{ i }}_delta_z: 0.0
+    {% endfor %}
+
+    # photoz shifts for the lenses
+    {% for i in range(10) %}
+    lens{{ i }}_delta_z: 0.0
+    {% endfor %}
+
+  two_point: &two_point
+    module: firecrown.ccl.two_point
+    sacc_file: &sacc_file syndata/srd_y10_3x2.sacc
+    fsky: *fsky
+    add_noise: *add_noise
+    parameters: *parameters #same as generation
+    kmax: *kmax
+    sources: &two_point_sources
+      {% for i in range(5) %}
+      src{{ i }}:
+        kind: WLSource
+        sacc_tracer:  src{{ i }}
+        Nz_type: ConstantDensGauss
+        Nz_nbins: 5
+        Nz_bin: {{ i }}
+        Nz_alpha: 0.68
+        Nz_z0: 0.11
+        Nz_sigmaz: 0.05
+        ellipticity_error: 0.26
+        number_density: 5.4 # per arcmin2 (27 arcmin-2 divided by 5 bins)
+        ia_bias: A0
+        systematics:
+          - delta_z_src{{ i }}
+      {% endfor %}
+      # Using 48 arcmin^-2 and multiplying times the integral of the bin's N(z) relative to the total
+      {% for i, ndens in [(0, 2.1), (1, 3.2), (2, 4.2), (3, 4.9),
```

Comment:
Again, some automated way to specify these 2.1, 3.2, etc would be nice. Something that user does not need to calculate them manually every time...